drivers/storage/rackspace/storage/rackspace_storage.go

```diff
@@ -0,0 +1,946 @@
+// /*
+// StorageDriver is a libStorage driver used by the routes to implement the
+// backend functionality.
+// Functions that inspect a resource or send an operation to a resource should
+// always return ErrResourceNotFound if the acted upon resource cannot be found.
+// */
+
+package storage
+
+import (
+	//"fmt" //DEBUG
```

Comment:
Hi @akiraw,

Please remove all unused imports. If you're using Atom you could use the go-fmt plug-in and set it to use the go-imports setting which handles this action for you. Thanks!